kubernetes_asyncio/watch/watch.py

```diff
@@ -70,6 +70,8 @@ def unmarshal_event(self, data: str, response_type):
         """
         js = json.loads(data)
 
+        if 'object' not in js:
```

Comment:
The 'type' is also required (at line 85), can you consider adding it to this condition? Thanks.

Reply:
Done!

Reply:
Thank you.